.hintrc

```diff
@@ -1,18 +0,0 @@
-{
-    "connector": {
-      "name": "local",
-      "options": {
-        "pattern": ["**", "!.git/**", "!node_modules/**"]
-      }
-    },
-    "extends": ["development"],
-    "formatters": ["stylish"],
-    "hints": [
-      "button-type",
-      "disown-opener",
-      "html-checker",
-      "meta-charset-utf-8",
-      "meta-viewport",
-      "no-inline-styles:error"
-    ]
```

Comment:
- [ ] Kindly restore all linters files that you deleted or reconfigure the linters for this project. It is a requirement for all projects that you will execute in Microverse. 👍 💻 

#### Hint: To configure linters for this project, kindly follow the instructions in the linked pasted below:

https://github.com/microverseinc/linters-config/tree/master/html-css

Reply:
For me in the my Mac computer is all working!